src/widget/wcoverartlabel.h

```diff
@@ -15,7 +15,7 @@ class WCoverArtLabel : public QLabel {
     Q_OBJECT
   public:
     explicit WCoverArtLabel(QWidget* parent = nullptr);
-    ~WCoverArtLabel() override;
+    ~WCoverArtLabel() override = default;
```

Comment:
This is redundant.

Reply:
No. The `override` keyword checks that the destructor of the base class is actually `virtual`, even if we use the default implementation. We already applied this technique multiple times.

Reply:
Ok